src/main/resources/assets/botania/lang/zh_CN.lang

```diff
@@ -478,13 +523,13 @@ achievement.botania:enchanterMake=Odds and Ends
 achievement.botania:enchanterMake.desc=制作一个魔力附魔台来更好的控制附魔
 achievement.botania:pollidisiacPickup=Romeo and Cinderella
 achievement.botania:pollidisiacPickup.desc=做一个畜牧花，在空气中制造爱情立场
-achievement.botania:brewPickup=Childhood Blues
+achievement.botania:brewPickup=オサナナブルー
 achievement.botania:brewPickup.desc=制作一个植物酿造台并用他酿造所有种类的药水
-achievement.botania:terraformRodCraft=World is Mine
+achievement.botania:terraformRodCraft=ワールドイズマイン
 achievement.botania:terraformRodCraft.desc=制作一把泰拉高并用他改变世界地形
 achievement.botania:bubbellPickup=深海少女
 achievement.botania:bubbellPickup.desc=制作一株气泡铃来方便水下建筑
-achievement.botania:luminizerRide=World's End Dancehall
+achievement.botania:luminizerRide=ワールズエンド ダンスホール
 achievement.botania:luminizerRide.desc=制作一些微光并用它来进行传输
```

Comment:
Well, that guideline is in early WIP, and that is **not** mandatory. Not quite sure if that guideline was carefully elaborated in [_Translation in a Nutshell_](https://www.youtube.com/watch?v=Oc319isHgNs&list=PLMqBx9PvgWaDhx0TFsxeDJVERdzk26PNL&index=2), but anyway, leaving English won't let people realise they are vocaloid stuff easily. That friend of mine who realized the fact of vocaloid reference did spend couple of days to search on google, niconico and even [bilibili](https://en.wikipedia.org/wiki/Bilibili) (#RepostEverywhere) to make a chart for it.